static/app/views/releases/detail/commitsAndFiles/filesChanged.tsx

```diff
@@ -117,12 +117,11 @@ class FilesChanged extends AsyncView<Props, State> {
               </PanelHeader>
               <PanelBody>
                 {files.map(filename => {
-                  const {authors} = repoData[filename];
                   return (
                     <StyledFileChange
                       key={filename}
                       filename={filename}
-                      authors={Object.values(authors)}
+                      authors={Object.values(repoData[filename].authors ?? {})}
```

Comment:
do we need 
```suggestion
                      authors={Object.values(repoData[filename]?.authors ?? {})}
```

if it's undefined?

Reply:
maybe it won't be?

Reply:
Yup, correct! Nice catch